addons/escoria-core/game/core-scripts/esc/types/esc_command.gd

```diff
@@ -82,26 +82,32 @@ func _init(command_string):
 
 # Check, if conditions match
 func is_valid() -> bool:
-	var command_found = false
-	for base_path in ProjectSettings.get("escoria/main/command_directories"):
-		var command_path = "%s/%s.gd" % [
-			base_path.trim_suffix("/"),
-			self.name
-		]
-		if ResourceLoader.exists(command_path):
-			command_found = true
-			
-	if not command_found:
+	if not command_exists():
+		return false
```

Comment:
This should probably go after the log output in the next few lines

Reply:
Oops, indeed. Fixed